d3-chord/d3-chord-tests.ts

```diff
@@ -15,10 +15,10 @@ import { ascending } from 'd3-array';
 // Preparatory Steps
 // ---------------------------------------------------------------------
 
-let context: CanvasRenderingContext2D;
+let context: CanvasRenderingContext2D | null= document.querySelector('canvas')!.getContext("2d");
```

Comment:
I fixed the occurrence of `"` above, will push as follow-up. Generally, I used single-quote as my go to for the definitions. I agree that when we have the "functional" changes to the definitions out of the way, we could now resort to the new `tslint` implementation on the types-2.0 branch. I.e. we drop a `tslint.json` into each definition folder, extend the defaults and add a quotation mark  rule to it.